src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -271,12 +271,25 @@ private SentenceCalcDates emptySentenceCalcDates(final Long bookingId) {
     @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA"})
     public PrivilegeSummary getBookingIEPSummary(final Long bookingId, final boolean withDetails) {
         final var offenderBooking = offenderBookingRepository.findById(bookingId).orElseThrow(EntityNotFoundException.withId(bookingId));
-        return offenderBooking.getIepSummary(withDetails).orElse(
-            PrivilegeSummary.builder()
+        final var iepSummary = offenderBooking.getIepSummary(withDetails);
+
+        if (iepSummary.isPresent()) {
+            return iepSummary.get();
+        } else {
+            final var iepDefault = availablePrisonIepLevelRepository.findByAgencyLocation_IdAndDefaultIep(offenderBooking.getLocation().getId(), true)
```

Comment:
Previously this would default the IEP level to `Entry` which was configured in Prison API in 2017 and not a valid level, this changes it to find the default IEP by prison